utils/utils.c

```diff
@@ -280,12 +281,87 @@ void fetchGlfsVolServerFromEnv()
   LOG("mgmt", GB_LOG_INFO, "Block Hosting Volfile Server Set to: %s", gbConf.volServer);
 }
 
+
 static int
-initLogDirAndFiles(char *newLogDir)
+glusterLogrotateConfigSet(const char *dom, char *logDir)
+{
+  char *buf = NULL, *line = NULL, *p;
+  int ret, m, len;
+  size_t n;
+  FILE *fp;
+
+  fp = fopen(GB_LOGROTATE_PATH, "r+");
+  if (fp == NULL) {
+    ret = -errno;
+    LOG(dom, GB_LOG_ERROR, "Failed to open file '%s', %m\n", GB_LOGROTATE_PATH);
```

Comment:
No '\n' needed here and other places too.

Reply:
Will fix it.
Thanks.